.changes/preview-comments.md

```diff
@@ -0,0 +1,5 @@
+---
+"action": patch
```

Comment:
We can call this a minor as well. I would qualify comments as a big enough feature.
```suggestion
"action": minor
```